packages/notification-center/src/i18n/languages/te.ts

```diff
@@ -0,0 +1,11 @@
+import { ITranslationEntry } from '../lang';
+
+export const TE: ITranslationEntry = {
+  translations: {
+    notifications: 'నోటిఫికేషన్‌లు',
+    markAllAsRead: 'అన్నీచదివినట్లుగా సూచించు',
```

Comment:
Reversing with Google translate, seems not to be very accurate. Missing the `all` it seems.

Reply:
No, it's there